plugin.program.autowidget/resources/lib/common/utils.py

```diff
@@ -23,16 +23,16 @@
     from urlparse import unquote
 
 try:
-    translate_path = xbmcvfs.translatePath
+    translatePath = xbmcvfs.translatePath
```

Comment:
This needs to either be changed *back*, or changed through the rest of the codebase to match here.